.github/workflows/build.yml

```diff
@@ -45,15 +45,14 @@ jobs:
         env:
           cache-name: cache-node-modules
         with:
-          path: ~/.npm
```

Comment:
This is the culprit right here. This does function as a global download cache for linux, and macos, but it was hard to find documentation on it. Every guide I looked at said to cache `node_modules` instead. 
On windows restoring this path restored less than 1 MB, where as for Linux and MacOS they were restoring between 25MB and 30MB.